src/js/components/LongDashPlaceholder.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+/**
+ * Component to display long dash placeholder
+ *
+ * @class LongDashPlaceholder
+ * @extends {React.Component}
+ * @returns {String} unicode dash
+ */
+class LongDashPlaceholder extends React.Component {
```

Comment:
`export default LongDashPlaceHolder = props => (<span>{"\u2014"}</span>);`

Reply:
I wrote like that first but then decided to follow the rest of our components, also is there any gain on this?

Reply:
No gain as far as I know. We have have some though - like `ConfigurationMapLabel`.

Reply:
Functional components will eventually be more performant (there are plans to improve the rendering of stateless components) but at the moment they're internally converted to _class_ based components which means there is little to no gain. Though one can use functional stateless components like any other function which removes the entire react create element step and improves performance a lot. For details please see: https://medium.com/missive-app/45-faster-react-functional-components-now-3509a668e69f

Reply:
In the future will make sense to use them whenever needed but I'm talking about our case today and like I said in my first comment above functional component was my first option but I assumed someone was going to complain of not following the patterns we have in the code base for the other components since the majority is written using `class` so I went with `class`.